testdata/configtest.json

```diff
@@ -2430,6 +2430,70 @@
      "supportedCurrencies": ""
     }
    ]
+  },
+  {
+   "name": "FTX",
+   "enabled": true,
+   "verbose": false,
+   "httpTimeout": 0,
+   "websocketResponseCheckTimeout": 0,
+   "websocketResponseMaxLimit": 0,
+   "websocketTrafficTimeout": 0,
+   "websocketOrderbookBufferLimit": 0,
+   "baseCurrencies": "",
+   "currencyPairs": {
+    "requestFormat": {
+     "uppercase": true
+    },
+    "configFormat": {
+     "uppercase": true
+    },
+    "useGlobalFormat": true,
```

Comment:
FTX doesn't use a global pair config format, please rid

Reply:
ive set it to false for now, but by rid u mean cya m88888888? just delete it n it automatically sets to false?

Reply:
Rid the whole block as like: https://github.com/thrasher-corp/gocryptotrader/blob/master/config_example.json#L539. You can do this manually or by running GCT with config=testdata/configtest.json (and also config_example.json) save the config via CTRL+C. Then ensure that you don't override any default values (ntp client, encrypt config prompt).